libs/sdk-ui-tests-e2e/cypress/integration/01-sdk-ui-dashboard/drillDown.spec.ts

```diff
@@ -337,6 +346,64 @@ describe("Drilling", () => {
                 ]);
             api.setUpDrillDownAttribute(DISPLAYFORM_PRODUCT);
         });
+
+        it("Drilling down on chart two drillable attribute separates", () => {
```

Comment:
in graphene test we use 2 charts: `bullet` and `column`, what type of chart do you use and how about the last one ?

Reply:
I don't want to create too many MD, So I select only 1 type chart for each testing.
Using bullet chart to test Drilling down on chart that have two attribute same bucket with two drillable attribute and using column chart to test Drilling down on charts that have two drillable attributes separates